src/node_modules/app/components/AccessibilityMenu.svelte

```diff
@@ -1,211 +1,32 @@
-	);
-	const cycleColorModels = makeCycleUsingAndStoreProp(
-		colorModels,
-		colorCorrectionOptions,
-		'colorModel'
-	);
-	const cycleHues = makeCycleUsingAndStoreProp(
-		hues,
-		colorCorrectionOptions,
-		'hue-rotate'
-	);
-	const cycleGrayscales = makeCycleUsingAndStoreProp(
-		grayscales,
-		colorCorrectionOptions,
-		'grayscale'
-	);
-	const cycleBrightnesses = makeCycleUsingAndStoreProp(
-		brightnesses,
-		colorCorrectionOptions,
-		'brightness'
-	);
-	const cycleContrasts = makeCycleUsingAndStoreProp(
-		contrasts,
-		colorCorrectionOptions,
-		'contrast'
-	);
-	const cycleSimulations = makeCycleUsingAndStoreProp(
-		simulations,
-		colorCorrectionOptions,
-		'simulate'
-	);
-
-	const toggleActive = () => {
-		const ccOptions = get(colorCorrectionOptions);
-		ccOptions.active = !ccOptions.active;
-		colorCorrectionOptions.set(ccOptions);
-	}
-
-	onMount( () => {
-		cycleFont();
-		cycleLineHeight();
-		cycleWordSpacing();
-		cycleLetterSpacing();
-	})
 </script>
 
-<div>
-	<header>Fonts and Text</header>
-	<button on:click={cycleFont}>Font: {$fontFamily}</button>
-	<button on:click={cycleSize}>Scale: {$fontScaling}</button>
-	<button on:click={cycleLineHeight}>Line height: {$lineHeight}</button>
-	<button on:click={cycleLetterSpacing}>Letter spacing: {$letterSpacing}</button>
-	<button on:click={cycleWordSpacing}>Word spacing: {$wordSpacing}</button>
-	<button disabled={!variationsEnabled} on:click={cycleVariation}>Settings: {$fontVariationSettings}</button>
-	<header>Color Vision Deficiencies</header>
-	<button on:click={toggleActive}>Active: {$colorCorrectionOptions.active}</button>
-	<button on:click={cycleSimulations}>Simulate: {$colorCorrectionOptions.simulate}</button>
-	<button on:click={cycleInverts}>Invert: {$colorCorrectionOptions.invert}%</button>
-	<button on:click={cycleColorModels}>Model: {$colorCorrectionOptions.colorModel}</button>
-	{#if $colorCorrectionOptions.colorModel === 'hue'}
-		<button on:click={cycleHues}>Hue: {$colorCorrectionOptions['hue-rotate']}deg</button>
-	{:else if $colorCorrectionOptions.colorModel === 'grayscale'}
-		<button on:click={cycleGrayscales}>Grayscale: {$colorCorrectionOptions.grayscale}%</button>
-	{/if}
-	<button on:click={cycleBrightnesses}>Brightness: {$colorCorrectionOptions.brightness}%</button>
-	<button on:click={cycleContrasts}>Contrast: {$colorCorrectionOptions.contrast}%</button>
+<div class={screen?.classes}>
 </div>
 
 <style>
 	div {
-		position: fixed;
-		top: 48px;
-		left: 48px;
+		display: grid;
+		width: 100%;
+		height: 64px;
 		background: white;
 		border: thin solid black;
 		z-index: 1;
-	}
-	button {
-		display: block;
-	}
-	header {
+		grid-template-areas: "indicators proppanel nav";
+		grid-template-columns: min-content 1fr min-content;
 		font-family: sans-serif;
 		font-size: 16px;
+		line-height: 24px;
+		letter-spacing: 0;
+		word-spacing: 0;
+	}
+	div.medium {
+		position: fixed;
+		width: 820px;
+		bottom: 64px;
+		left: 50%;
+		margin-left: -410px;
```

Comment:
I'd use:
```suggestion
	width: 50%;
	bottom: 100px;
	left: 25%;
```

with a `min-width` value for safety, and so that it's well distanced from the timeline top.

Possibly, when we'll have content in next PRs I'd make it responsive, to maximise usability:
- with large screens, having it too wide forces you to move the pointer a lot to select different radios
- with small screens, we might need to go for `width: 60%;` or more to make it readable